### ISSUE TITLE
Dropdown not accessible with arrow keys

### ISSUE THREAD

**Issue by olilutz** [Dogfalo/materialize#3502]
In a dropdown I can't use the arrow keys to go the list up or down. In a mobile version this is not necessary but if I use a keyboard it would be so much easier and faster to use arrow keys.
Especially in the autocomplete dropdown I always have to switch from typing to select the correct option with mouse and click on it in case I don't want the first option entered.
It works fine in the material select, everything is already in the js-code just needs to be adapted.
Maybe it could be fixed in combination with #3486.

**Comment by nuplo** [comment]
hey, today I added to autocomplete arrow keys usage and some other function https://github.com/Dogfalo/materialize/pull/3694

**Comment by mishalmsb** [comment]
Hey, any news on using arrow on autocomplete?
Thanks

**Comment by Dogfalo** [comment]
Unlikely to be added before v1

**Comment by ChildishGiant** [comment]
Working on  0.100.1 and up. Doesn't scroll to the current selection though.

**Comment by Dogfalo** [comment]
Added in v1-dev

**Comment by ChildishGiant** [comment]
How do we test out this branch?

**Comment by Dogfalo** [comment]
You can `git checkout v1-dev` and run grunt monitor to build it
